### DISCUSSION TITLE
Desktop Layout nach Smartphone Layout kopieren

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#2158] [Q&A] [answered]
Hallo,

lässt sich das Desktop Layout zu dem Smartphone Layout kopieren?

**Comment by mcuiobroker** [comment, marked as answer]
Für v3 ja, für v3.1.2 muss das Programm angepasst werden. 
In der Doku desktosmart.
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/system-effektprog./v3-systemprogramme/copydesktosmart

Ansonsten kann man es manuell kopieren.
Mit einem Editor.
Widgets -> widgetsDesktop Inhalt nach widgetsSmartphine kopieren.
Dannnwieder in widgets DP einfügen.
